packages/rules-unit-testing/src/api/index.ts

```diff
@@ -160,7 +160,7 @@ export type FirebaseEmulatorOptions = {
 
 function trimmedBase64Encode(val: string): string {
   // Use base64url encoding and remove padding in the end (dot characters).
-  return base64Encode(val).replace(/\./g, "");
+  return base64Encode(val).replace(/\./g, '');
```

Comment:
Okay, somehow this file is not properly formatted. Can we address this in a different PR though? We're not actually changing RUT here.

Reply:
Ah, the works of an overly aggressive VSCode formatting, undoing and will send out a separate PR!